now.json

```diff
@@ -6,14 +6,13 @@
     "public": true,
     "builds": [
         { "src": "public/*", "use": "@now/static" },
-        { "src": "package.json", "use": "@now/static-build" },
+        { "src": "package.json", "use": "@now/static-build", "config": { "distDir": "public/dist" } },
```

Comment:
I think you can change `public/dist` to `public`.

And if you could change the `now-build` step to only emit `browser.js` that would be awesome!

Reply:
Will do, working on it. ;)